numcodecs/tests/common.py

```diff
@@ -12,7 +12,7 @@
 
 
 from numcodecs.compat import buffer_tobytes, ndarray_from_buffer
-from numcodecs import *  # flake8: noqa
+from numcodecs.registry import get_codec
```

Comment:
Think this might be related to the CI failure. Not sure though.

Reply:
Yep, my bad. Should be fixed now.